### PR TITLE
chore(release): Revise v6.2.1 release notes for readability

### PR DESCRIPTION
**Pull request by SyntaxColoring**
# Overview

Revisions to the v6.2.1 release notes to make them more concise and readable. Incorporates @ecormany's suggestions from #12044.

# Review requests

words good?

### CODE REVIEW

File: api/release-notes.md [hunk 1]
```diff
@@ -12,11 +12,10 @@ Welcome to the v6.2.1 release of the OT-2 software! This hotfix release addresse
 
 ### Bug Fixes
 
+- When you upload a protocol or set up a run, the OT-2 is now less likely to show connection errors.
```
Comment (SyntaxColoring) [inline api/release-notes.md:15]: Technically, in some cases, it will also respond faster. I don't know if that's worth including?

Comment (ecormany) [reply api/release-notes.md:15]: i think stating the cause is more important (big protocol files, either in transit or at rest). even though the threshold for "big" is quite small by modern standards, this wasn't an issue across _all_ uploads or runs.

**Review by ecormany — APPROVED** [review]

File: api/release-notes.md (hunk 1 above)
Comment (SyntaxColoring) [reply api/release-notes.md:15]: Small/normal protocols were an issue, in a sense. If you had 20 runs that collectively added up to something big, that could cause performance problems and disconnections the first time you did anything protocol-related with the robot. Even if no individual protocol was large.

Separately, individual large protocols *also* caused performance and disconnection problems, but in different places.

Comment (SyntaxColoring) [reply api/release-notes.md:15]: Leaving this as-is.

**Pull request merged**